src/components/CodeEditor.tsx

```diff
@@ -50,7 +52,15 @@ export const CodeEditor = ({
 
   const [files, setFiles] = useState<Record<string, string>>({
     "index.tsx": initialCode,
-    "manual-edits.json": "",
+    "manual-edits.json": JSON.stringify(
+      {
+        pcb_placements: [],
+        edit_events: [],
+        manual_trace_hints: [],
+      },
+      null,
+      2,
+    ),
```

Comment:
this eventually needs to be loaded from the snippet, but this is a great mergable start!